Yosemite/Yosemite/Stores/TaxStore.swift

```diff
@@ -98,12 +99,28 @@ private extension TaxClassStore {
             }
         }
     }
+
+    func retrieveTaxRates(siteID: Int64,
+                          pageNumber: Int,
+                          pageSize: Int,
+                          onCompletion: @escaping (Result<([TaxRate]), Error>) -> Void) {
```

Comment:
I think this is a typo, since we're using a tuple with the array as its only value:
```suggestion
                          onCompletion: @escaping (Result<[TaxRate], Error>) -> Void) {
```

Reply:
Yeah, done in 36aa52386d22ecb32fa8acdc895675c88b2d483e